src/main/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateService.kt

```diff
@@ -52,6 +53,7 @@ class AssessmentUpdateService(
       throw UpdateClosedEpisodeException("Cannot update a closed or completed Episode ${episode.episodeUuid} for assessment ${episode.assessment.assessmentUuid}")
 
     episode.updateEpisodeAnswers(updatedEpisodeAnswers)
+    episodeService.removeOrphanedAnswers(episode)
```

Comment:
This would be to remove answers that are no longer relevant, such as where we answer `yes+details` and then change this to `no` it removes the `details` answer from the database

Reply:
As discussed - we'e required to do this on both update/clone